client/client_update.go

```diff
@@ -108,16 +108,15 @@ type UpdateResponse struct {
 	Image struct {
 		URI      string
 		Checksum string
-		YoctoID  string `json:"yocto_id"`
-		ID       string
+		Name     string `json:"artifact_name"`
```

Comment:
I know there is a separate task for this, but will this be as described here: https://tracker.mender.io/browse/MEN-734?